addon/mixins/time-series-labeler.js

```diff
@@ -178,7 +171,49 @@ export default Ember.Mixin.create({
 
   //  This is the set of ticks on which labels appear.
   labelledTicks: Ember.computed('unfilteredLabelledTicks', 'tickFilter', function() {
-    return this.get('unfilteredLabelledTicks').filter(this.get('tickFilter'));
+    var ticks = this.get('unfilteredLabelledTicks');
+    return ticks.filter(this.get('tickFilter'));
+  }),
```

Comment:
Not sure this needed changing. If we did need to change the actual computations, though, I would support splitting them into two lines as well...